show/interfaces/__init__.py

```diff
@@ -118,6 +118,48 @@ def description(interfacename, namespace, display, verbose):
 
     clicommon.run_command(cmd, display_cmd=verbose)
 
+#
+# 'txerror' subcommand ("show interfaces tx_error")
+#
+@interfaces.command('tx_error')
+@click.argument('interfacename', required=False)
+def tx_error(interfacename):
+    """Show Interface tx_error information"""
+
+    state_db = SonicV2Connector(host='127.0.0.1')
+    state_db.connect(state_db.STATE_DB, False)   # Make one attempt only
+    prefix_statedb = "TX_ERR_STATE|"
```

Comment:
The table name should come from pyswsscommon library. Please do not use hardcoded values